app/Exceptions/Handler.php

```diff
@@ -72,7 +72,6 @@ protected function buildExceptionContext(Throwable $e): array
                     unset($params['p']);
                 }
             } elseif (str_contains($context['url'], '/API/')) {
-                unset($params['z']);
```

Comment:
We should probably still exclude this from the exception log, as consumers will likely continue to pass it for quite some time.

Reply:
Yeah, that's a good point. Will revert.